alts/worker/tasks.py

```diff
@@ -124,6 +132,9 @@ def is_success(stage_data_: dict):
             response = requests.post(
                 full_url, json=payload,
                 headers={'Authorization': f'Bearer {CONFIG.bs_token}'})
-            response.raise_for_status()
+            try:
```

Comment:
That's bad for celery tasks status. This will result task status being "SUCCESS" while it's not

Reply:
Done